src/gps/GPS.cpp

```diff
@@ -502,11 +503,22 @@ void GPS::setGPSPower(bool on, bool standbyOnly, uint32_t sleepTime)
             }
             msglen = gps->makeUBXPacket(0x02, 0x41, 0x08, gps->_message_PMREQ);
             gps->_serial_gps->write(gps->UBXscratch, msglen);
+        } else if (gnssModel == GNSS_MODEL_MTK) {
+            // Enter soft standby mode
+            LOG_DEBUG("L76K entering soft standby mode\n");
+            gps->_serial_gps->write("$PMTK161,0*28\r\n");
```

Comment:
@neilhao this command seems correct based on the MTK datasheets and I see that SoftRF does the same thing. However, I'm not seeing any power savings with my ATGM336H module. Do you happen to know if they implement the standby command? I can't find any docs specific to the module.

Reply:
Give "$PCAS12,65535*1E\r\n" a try on your personal branch @thebentern @neilhao if this command doesn't work for you. It's listed in the CASIC protocol manual as a standby control msg. Should make the receiver standby for 65535 seconds, if the module supports it.